setup.cfg

```diff
@@ -135,7 +135,7 @@ install_requires=
    beautifulsoup4>=4.8
    html5lib>=0.999
    keyring>=15.0
-   pyvo>=1.1
+   pyvo>=1.5
```

Comment:
pyvo 1.5 isn't available yet - this is way too big a jump forward in requirements

Reply:
I'm pushing to have that `PyVO` released soon as it's about time to do. We need it for quantities in the enhanced table.

Reply:
this is a really big leap in required versions, especially if it's only required for alma. 
(I suppose we'll need some of the SIA2 fixes in the 1.4.x series for irsa, too, but still 1.5 feels too strict a requirement)

Maybe, for now, do a version check in the module instead?

Reply:
Yeah, I feel strongly about this: a big version jump should be in-module only, not an install-level requirement

Reply:
there will be a lot of updates in pyvo that we'll rely on here (around download and handling cloud data), so at some point we'll need to bite the bullet

Reply:
That's fine.  The required version should at least be released before we consider taking that jump, though.

Reply:
I can easily isolate the 1.5 dependency in alma but it's just annoying that some users will have to install it manually (it's just the quality feature). 

`1.4` has been out for a while and probably already used by the `astroquery` community. We haven't bumped the dependency here because the code did not depend on any new `1.4` API. 

We obviously need to release `1.5` first and then can wait for a bit before merging this draft.

Reply:
Let's have a separate PR for updating the pyvo version requirement, and we can just make this PR depend on that one

Reply:
(we shouldn't be having discussions like this in a side comment on an unrelated PR... hard for people to find if it becomes important)

Reply:
let's aim to have a tagged 0.4.7 out, and then update dependencies. I want to drop some ancient python and astropy support as we don't need to drag those along indefinitely. I'm still on the fence of mandating to have a latest and greatest of something, so we may likely sit on requiring 1.5 pyvo for a while (but we can certainly upgrade the install instructions in a way that it would help users to upgrade the dependencies for them)